faunadb/errors.py

```diff
@@ -103,18 +100,24 @@ class ErrorData(object):
   @staticmethod
   def from_dict(dct):
     code = get_or_invalid(dct, "code")
-    return ValidationFailed(dct) if code == "validation failed" else ErrorData(dct, code)
+    description = get_or_invalid(dct, "description")
+    position = dct.get("position")
+    if code == "validation failed":
+      failures = [Failure.from_dict(_) for _ in get_or_invalid(dct, "failures")]
```

Comment:
Since we aren't throwing away the value here, could you rename `_` to something more descriptive?